bin/varnishncsa/varnishncsa.c

```diff
@@ -916,7 +916,8 @@ frag_line(int force, const char *b, const char *e, struct fragment *f)
 }
 
 static void
-process_hdr(const struct watch_head *head, const char *b, const char *e)
+process_hdr(int force, const struct watch_head *head, const char *b,
+    const char *e)
```

Comment:
The commit introducing the force flag is immediately undone by the next commit introducing the format policy, you can remove it entirely from the patch series.